app/lib/common/providers/space_providers.dart

```diff
@@ -233,38 +228,27 @@ final spaceRelationsOverviewProvider =
   if (relatedSpaces == null) {
     throw SpaceNotFound;
   }
-  final membership = await ref.watch(roomMembershipProvider(spaceId).future);
-  bool hasMoreSubspaces = false;
-  bool hasMoreChats = false;
-  final List<Space> knownSubspaces = [];
-  final List<Convo> knownChats = [];
+  bool hasMore = false;
+  final List<String> knownSubspaces = [];
+  final List<String> knownChats = [];
   final List<String> suggested = [];
   List<Space> otherRelated = [];
-  for (final related in relatedSpaces.children()) {
+  final children = relatedSpaces.children();
+  for (final related in children) {
     String targetType = related.targetType();
     final roomId = related.roomId().toString();
     if (related.suggested()) {
       suggested.add(roomId);
     }
-    if (targetType == 'ChatRoom') {
-      try {
-        final chat = await ref.watch(chatProvider(roomId).future);
-        knownChats.add(chat);
-      } catch (e) {
-        hasMoreChats = true;
-      }
+    if (targetType == 'Unknown') {
+      // this one is not found locally
+      hasMore = true;
+    } else if (targetType == 'ChatRoom') {
+      // we know this as a chat room
+      knownChats.add(roomId);
```

Comment:
This is a first part of the fix:
`ChatRoom` and `Space` are only given  `targetTypes` when we can _load_ them rooms and check them for it. For the remote rooms, this isn't the case, hence the `Unknown` flag. Previously, however, we'd only flag the `hasMoreChat` if we had tried and failed to read a room. 

The fix here is to not guess the type, but generalize that `hasMore` can be either chats or spaces until we have fetched them from remote.